compiler/luci-interpreter/src/kernels/TestUtils.h

```diff
@@ -58,6 +60,113 @@ template <typename T> std::vector<T> extractTensorData(const Tensor &tensor)
 std::vector<::testing::Matcher<float>> ArrayFloatNear(const std::vector<float> &values,
                                                       float max_abs_error = 1.0e-5f);
 
+template <typename T>
+inline std::vector<T> quantize(const std::vector<float> &data, float scale, int32_t zero_point)
+{
+  std::vector<T> q;
+  for (const auto &f : data)
+  {
+    if (getElementType<T>() == DataType::FLOAT32)
+      q.push_back(f);
+    else
+      q.push_back(static_cast<T>(std::max<float>(
+          std::numeric_limits<T>::min(),
+          std::min<float>(std::numeric_limits<T>::max(), std::round(zero_point + (f / scale))))));
+  }
+  return q;
+}
+
+template <typename T>
+inline std::vector<float> dequantize(const std::vector<T> &data, float scale, int32_t zero_point)
+{
+  std::vector<float> f;
+  for (const T &q : data)
+  {
+    if (getElementType<T>() == DataType::FLOAT32)
```

Comment:
Similar to the above comment (replacing with assert).

Reply:
Okay, I fix like what you suggested.